src/__experimental__/components/radio-button/radio-button.stories.js

```diff
@@ -4,47 +4,28 @@ import {
   boolean, text, number, select
 } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
 import { dlsThemeSelector, classicThemeSelector } from '../../../../.storybook/theme-selectors';
 import OptionsHelper from '../../../utils/helpers/options-helper';
 import { RadioButton, RadioButtonGroup } from '.';
-import { info, notes } from './documentation';
+import { info, infoValidations, notes } from './documentation';
 
-function makeStory(name, themeSelector) {
-  const component = () => {
-    const knobs = defaultKnobs();
+const trueBool = true;
```

Comment:
What is the purpose of this?

Reply:
`trueBool` should be removed and `true` used instead. Not sure why this was introduced.

Reply:
it's used only once, so we'll never know what the author had in mind...
changed